infra/resources/_modules/cdn/dns.tf

```
@@ -17,4 +17,28 @@ resource "azurerm_dns_txt_record" "wallet_io_pagopa_it" {
     value = "qidvirtenu"
   }
   tags = var.tags
+}
+
+# TXT for cbor Maven namespace verification
+resource "azurerm_dns_txt_record" "cbor_wallet_io_pagopa_it" {
+  name                = "@"
```

I think the name of the record should coincide with the namespace (cbor.wallet.io.pagopa.it)

```suggestion
  name                = "cbor"
```